search/includes/classes/class-search.php

```diff
@@ -497,20 +497,25 @@ protected function setup_constants() {
 	 */
 	protected function maybe_enable_ep_query_logging() {
 		add_action( 'plugins_loaded', [ $this, 'enable_ep_query_logging_if_query_monitor_enabled' ] );
+		add_action( 'plugins_loaded', [ $this, 'load_ep_debug_bar_panel' ] );
+	}
+
+	public function load_ep_debug_bar_panel() {
```

Comment:
This is incorrectly named - we don't have the Debug Bar anymore + it's not loading the panel but a single helper function.

Reply:
I renamed it to `load_ep_get_query_log_function`, what do you think. Feel free to drop name suggestions.